instrumentation/dropwizard-testing/src/test/groovy/DropwizardTest.groovy

```diff
@@ -122,7 +122,7 @@ class DropwizardTest extends HttpServerTest<DropwizardTestSupport> {
         "${SemanticAttributes.HTTP_USER_AGENT.key()}" TEST_USER_AGENT
         "${SemanticAttributes.HTTP_CLIENT_IP.key()}" TEST_CLIENT_IP
         // exception bodies are not yet recorded
-        "${SemanticAttributes.HTTP_RESPONSE_CONTENT_LENGTH.key()}" { it == responseContentLength || /* async */it == null }
+        "${SemanticAttributes.HTTP_RESPONSE_CONTENT_LENGTH.key()}" { it == 0 || it == responseContentLength || /* async */it == null }
```

Comment:
allowing 0 indicates that the response was not counted. if it's not counted in the tests I wouldn't expect it to be counted elsewhere.

Reply:
Well, it's to be expected that all frameworks that use `getOutputStream()` will now get a 0.

Reply:
This means it misses spring boot's default configuration which I'd hazard to say is a large portion of applications being instrumented. Is there some alternative implementation or is this attribute just being dropped?

Reply:
There were two:
* https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/1104 contained a separate instrumentation for servlet 3.0 and 3.1; it had lots of copy-pasted code
* In https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/1149 I initially tried to just copy the `WriteListener` class that was absent in 3.0, but that failed. Then I tried to generate a class extending `ServletOutputStream` in runtime, which probably should work -- but there's no way to load it correctly without modifying `Instrumenter.Default` heavily.

Reply:
@FrankSpitulski We're leaning towards just using `content-length` header for servlet. The counting approach seems to have too many corner cases for us to deal with well unfortunately 😿 

@mateuszrzeszutek I don't think we should set `0` if we couldn't count, `0` means no content, not content couldn't be counted - can you update the implementation of `getContentLength` to avoid setting `0` when `printWriter` is `null`?

Reply:
I've removed all the counting stuff altogether, according to https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/1167#issuecomment-689345608